docs/API/get-started.md

```diff
@@ -0,0 +1,37 @@
+Title: Get Started with Flotiq API Integration
+
+Description: Get started with integrating Flotiq API into your application.
+
+!!! note "Before you begin, make sure you have the following:"
+    * Flotiq Account: Sign up for an account on Flotiq website if you haven't already.
+    * API Key: Obtain your Application Read and Write API Key from the Flotiq dashboard. You'll need it to perform API operations.
+    * Basic understanding of RESTful APIs: Familiarize yourself with the concepts of RESTful APIs if you're new to this.
+
+#### Step 1: Creating Content Type Definitions
+To start using Flotiq API, you need to define your content types. Content Type represents a model of data defined within the Content Repository. There are two ways to create a Content Type:
+
+* Via API: You can programmatically create a Content Type by sending a properly formatted POST request to the 
+/api/v1/internal/contenttype endpoint. [Check it here](content-type/creating-ctd.md) to know more about it.
+
+* Using our Flotiq's content type editor: Flotiq provides a convenient Content Modeler tool within the platform for creating Content Types visually. You can access it through the Flotiq dashboard.
+
+#### Step 2: Implementing API Integration
+Once you have defined your Content Types, you can proceed with integrating Flotiq API into your application. Here's a high-level overview of the steps involved:
+
+* Authentication: Use your [API key](index.md) to authenticate API requests. Include the key in the request headers or as a query parameter, depending on the API endpoint.
+* Data Manipulation: Perform CRUD operations ([Create](content-type/creating-ctd.md), [Read](content-type/getting-ctd.md), [Update](content-type/updating-ctd.md), [Delete](content-type/deleting-ctd.md)) on your Content Types using the appropriate API endpoints and HTTP methods.
```

Comment:
I'd make this into a separate subsection and show basic operations based on a sample code. For me, getting started page should give you all the necessary info to actually get started without digging too much through documentation. E.g. link to the dashboard page where API key can be obtained (without too many details) and info on how to use it in requests.

We could even utilize the [mkdocs-include-markdown-plugin](https://github.com/mondeja/mkdocs-include-markdown-plugin) to have some snippets extracted into separate templates if it made them repeat too much.